src/executor/create_executor.cpp

```diff
@@ -139,7 +140,7 @@ bool CreateExecutor::CreateTable(const planner::CreatePlan &node) {
         // Sink Column Offsets
         std::vector<oid_t> sink_col_ids;
         for (auto col_name : fk.foreign_key_sinks) {
-          oid_t col_id = source_table->GetSchema()->GetColumnID(col_name);
+          oid_t col_id = sink_table->GetSchema()->GetColumnID(col_name);
```

Comment:
Source -> sink table change, is a bug fix?

Reply:
This is a bug fix